docs/visualize/timelion.asciidoc

```diff
@@ -433,3 +433,38 @@ image::images/timelion-conditional04.png[]
 {nbsp}
 
 For additional information on Timelion conditional capabilities, go to https://www.elastic.co/blog/timeseries-if-then-else-with-timelion[I have but one .condition()].
+
+[float]
+[[customizing-data-series-y-axis]]
+==== Customize data series Y axis
+
+It is possible to use different Y axis scales and ranges.
```

Comment:
I would plug in the data series part here. For example:

Use different y-axis scales and ranges for your data series.